docs/user/integrations.rst

```diff
@@ -1,5 +1,7 @@
-VCS Integrations
-================
+How To Integrate Your Git Platform
```

Comment:
We do support other VCS that are not Git, so I'm not 100% sure this title is correct.

Also, the page talks about "VCS providers", so if we are changing the title, we should also rewrite those sentences as well.

Reply:
Aborted